demos/gaze_estimation_demo/cpp_gapi/main.cpp

```diff
@@ -4,15 +4,14 @@
 
 #include <monitors/presenter.h>
 #include <utils/args_helper.hpp>
-#include <utils/slog.hpp>
```

Comment:
`slog` is still used in this demo, it should be directly included.

Reply:
And, it seems, I forgot to include it directly in the [smart_classroom_demo_gapi](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/demos/smart_classroom_demo/cpp_gapi/main.cpp), when modified it. Can you add it here also, please?

Reply:
Some files have dependencies with slog.hpp, m b, because this works fine.  This include is returned.

Reply:
face_inference_results.hpp contains this file for gaze_estimation
and
logger.hpp for smart_classroom

Reply:
Could the thread will be resolved?